src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -1627,6 +1648,14 @@ private native void pairDevice(
       NetworkCredentials networkCredentials,
       @Nullable ICDRegistrationInfo icdRegistrationInfo);
 
+  private native void pairDeviceThroughNfc(
```

Comment:
pairDevice above should become pairDeviceThoguhBle for consistency in naming.

we could also consider having a single pair device and passing in the type (nfc or ble) as an argument, however having constants in sync between JNI and java seems more painful.